frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -59,7 +59,7 @@ export const BrowseModels = () => {
     return { filteredModels };
   }, [actualModelFilters, doVerifiedModelsExist, models]);
 
-  const recentModelsResult = useListRecentItemsQuery(undefined, {
+  const recentModelsResult = useListRecentsQuery(undefined, {
     refetchOnMountOrArgChange: true,
```

Comment:
since this is a request for recent views, we can add `context=["views"]` here. Or I can do that in a separate PR